pymeasure/instruments/ametek/ametek7270.py

```diff
@@ -98,36 +101,45 @@ class Ametek7270(Instrument):
         validator=truncated_discrete_set,
         values=TIME_CONSTANTS,
         map_values=True,
-        check_set_errors=True
+        check_set_errors=True,
     )
 
     x = Instrument.measurement("X.",
-                               """ Reads the X value in Volts """
+                               """ Reads the X value in Volts """,

```

Comment:
I know, that you did not change the docstring itself, so you do not have to change it, but I want to raise your attention, that that docstring is not according to our coding style.

For your information: A merely gettable property should start with "Measure" or "Get", a settable one with "Set" and a property gettable and settable with "Control".
For more details, see https://pymeasure.readthedocs.io/en/latest/dev/coding_standards.html#docstrings

If you want to, you may improve the docstrings. In that case, I'd kindly ask you to remove the Ametek720 from the `grandfathered_docstring_instruments` in "tests/instruments/test_all_instruments.py" such that docstring tests will be performed on this instrument (I just saw, that it is twice in that list).

You might do that docstring rework in a second PR, if you want, to finish this content rework first.

Reply:
Ok I'll see that later